src/main/java/org/springframework/samples/petclinic/HelloWorld.java

```diff
@@ -0,0 +1,12 @@
+/* This is a simple Java program. 
+FileName : "HelloWorld.java". */
+class HelloWorld 
+{ 
+	// Your program begins with a call to main(). 
+	// Prints "Hello, World" to the terminal window. 
+	public static void main(String args[]) 
+	{ 
+		System.out.println("Hello, World"); 
```

Comment:
Hey, System.out.println is not accepted in the code and you need enable logging to this class